src/encoded/docs/schema-changes.md

```diff
@@ -154,44 +154,115 @@ This new object is an array of example objects that can successfully POST agains
 Updating an existing schema
 ----------------
 
-There are 2 categories when we talk about updating an existing schema:
+There are two situations we need to consider when updating an existing schema: (1) No update of the schema version (2) Update schema version
 
-* No affect on schema version
-* Update schema version
+### No change on schema version
 
+* Schema version should not to be updated if the change introduced is not going to cause a potential invalidation of the existing objects in the database. For example an addition of a new enum value to a list of existing enums can not cause schema invalidation, but will simply extend the list of potential values to choose from.
 
-**The schema version is updated, requiring an upgrade of the schema, when the objects stored in the database will not validate and POST to the updated schema.**
+**Follow the steps as outlined below**
 
+1. In the **schemas** directory, edit the existing properties in the corresponding JSON file named after the object.
 
-### No affect on schema version
+2. In the **types** directory, make appropriate updates to object class by adding *embedding*, *reverse links*, and *calculated properties* as necessary.
 
-1. In the **schemas** directory, make edits  to properties in the JSON file named after the object.
+3. Update the inserts within the **data/inserts** directory.
 
-2. In the **types** directory, make appropriate updates to object class by adding *embedding*, *reverse links*, and *calculated properties* as necessary.
+4. Add test in the **tests** directory to make sure your schema change functions as expected. For example, if we included a minor change in treatment object such that *μg/kg* could be specified as treatment units, the following test should allow one to test whether the update has been successfully implemented or not:
```

Comment:
clearly separate the example from the instructions in step 4. It is easy to think that the treatment example is part of the step 4.